src/zen_qp.c

```diff
@@ -27,100 +27,99 @@
 #include <lua_functions.h>
 #include <zen_octet.h>
 
+/*
+  Quantum proof dilithium digital signature
+*/
 #define PQCLEAN_DILITHIUM2_CLEAN_CRYPTO_PUBLICKEYBYTES 1312
 #define PQCLEAN_DILITHIUM2_CLEAN_CRYPTO_SECRETKEYBYTES 2528
-#define PQCLEAN_DILITHIUM2_CLEAN_CRYPTO_BYTES 2420
-#define PQCLEAN_DILITHIUM2_CLEAN_CRYPTO_ALGNAME "Dilithium2"
-
-// Post quantum digital signature
+#define PQCLEAN_DILITHIUM2_CLEAN_CRYPTO_BYTES          2420
+#define PQCLEAN_DILITHIUM2_CLEAN_CRYPTO_ALGNAME        "Dilithium2"
 extern int PQCLEAN_DILITHIUM2_CLEAN_crypto_sign_keypair(uint8_t *pk, uint8_t *sk);
-
 extern int PQCLEAN_DILITHIUM2_CLEAN_crypto_pub_gen(uint8_t *pk, uint8_t *sk);
-
-extern int PQCLEAN_DILITHIUM2_CLEAN_crypto_sign_signature(
-    uint8_t *sig, size_t *siglen,
-    const uint8_t *m, size_t mlen, const uint8_t *sk);
-
-extern int PQCLEAN_DILITHIUM2_CLEAN_crypto_sign_verify(
-    const uint8_t *sig, size_t siglen,
-    const uint8_t *m, size_t mlen, const uint8_t *pk);
-
-extern int PQCLEAN_DILITHIUM2_CLEAN_crypto_sign(
-    uint8_t *sm, size_t *smlen,
-    const uint8_t *m, size_t mlen, const uint8_t *sk);
-
-extern int PQCLEAN_DILITHIUM2_CLEAN_crypto_sign_open(
-    uint8_t *m, size_t *mlen,
-    const uint8_t *sm, size_t smlen, const uint8_t *pk);
-
-// Post quantum cipher
+extern int PQCLEAN_DILITHIUM2_CLEAN_crypto_sign_signature(uint8_t *sig, size_t *siglen, const uint8_t *m, size_t mlen, const uint8_t *sk);
+extern int PQCLEAN_DILITHIUM2_CLEAN_crypto_sign_verify(const uint8_t *sig, size_t siglen, const uint8_t *m, size_t mlen, const uint8_t *pk);
+extern int PQCLEAN_DILITHIUM2_CLEAN_crypto_sign(uint8_t *sm, size_t *smlen, const uint8_t *m, size_t mlen, const uint8_t *sk);
+extern int PQCLEAN_DILITHIUM2_CLEAN_crypto_sign_open(uint8_t *m, size_t *mlen, const uint8_t *sm, size_t smlen, const uint8_t *pk);
+
+/*
+  Quantum proof kyber kem/cipher
+*/
 #define PQCLEAN_KYBER512_CLEAN_CRYPTO_SECRETKEYBYTES  1632
 #define PQCLEAN_KYBER512_CLEAN_CRYPTO_PUBLICKEYBYTES  800
 #define PQCLEAN_KYBER512_CLEAN_CRYPTO_CIPHERTEXTBYTES 768
 #define PQCLEAN_KYBER512_CLEAN_CRYPTO_BYTES           32
-#define PQCLEAN_KYBER512_CLEAN_CRYPTO_ALGNAME "Kyber512"
-#define KYBER_SSBYTES  32   /* size in bytes of shared key */
-
+#define PQCLEAN_KYBER512_CLEAN_CRYPTO_ALGNAME         "Kyber512"
+#define KYBER_SSBYTES                                 32   /* size in bytes of shared key */
 extern int PQCLEAN_KYBER512_CLEAN_crypto_kem_keypair(uint8_t *pk, uint8_t *sk);
-
 extern int PQCLEAN_KYBER512_CLEAN_crypto_pub_gen(uint8_t *pk, uint8_t *sk);
-
 extern int PQCLEAN_KYBER512_CLEAN_crypto_kem_enc(uint8_t *ct, uint8_t *ss, const uint8_t *pk);
-
 extern int PQCLEAN_KYBER512_CLEAN_crypto_kem_dec(uint8_t *ss, const uint8_t *ct, const uint8_t *sk);
 
+/*
+  Quantum proof Streamlined NTRU Prime kem/cipher
+*/
+#define PQCLEAN_SNTRUP761_CLEAN_CRYPTO_SECRETKEYBYTES  1763
```

Comment:
it makes sense to use this define in the zencode implementation
which in turn calls for the need of a header for zen_qp.h, about time for it, so many externs...